vllm/v1/attention/backends/flash_attn.py

```diff
@@ -489,7 +482,11 @@ def forward(
         # For decoder and cross-attention, use KV cache as before
         key_cache, value_cache = kv_cache.unbind(0)
 
-        if self.kv_sharing_target_layer_name is None:
+        # key and value may be None in the case of cross attention. They are
+        # calculated once based on the output from the encoder and then cached
+        # in KV cache.
+        if (self.kv_sharing_target_layer_name is None and (key is not None)
+                and (value is not None)):
```

Comment:
```suggestion
        if (self.kv_sharing_target_layer_name is None and key is not None and value is not None):
```
and use the formatter to keep the line shorter than the limit

Reply:
done, thanks!